src/all.test.js

```diff
@@ -149,4 +149,21 @@ describe('GOV.UK Frontend', () => {
 
     expect(functionCalls).toBeNull()
   })
+  describe('responsive typography', async () => {
+    it('is enabled by default', async () => {
```

Comment:
Rather than adding these, can we update the tests in `src/helpers/typography.test.js`? (We currently explicitly disable responsive mode, and then have a test for when it's enabled, but we should probably flip that logic?)